cli/cluster/node.py

```diff
@@ -17,17 +17,7 @@ def __init__(self, name, cloud):
         **kwargs:
```

Comment:
These changes are not required now. Please revert.

Reply:
Done, reverted the changed.